pypuppetdb/types.py

```diff
@@ -128,22 +176,36 @@ def __str__(self):
     def __unicode__(self):
         return self.__string
 
+    def events(self):
+        """Get all events for this report."""
+        return self.__api.events(query=self.__query_scope)
```

Comment:
This needs to be done also for `return self.__api.reports(self.__query_scope)` below